src/core/OptionPanelApi.ttslua

```diff
@@ -0,0 +1,16 @@
+do
+  local OptionPanelApi = {}
+
+  -- loads saved options
+  ---@param options Table New options table
+  OptionPanelApi.loadSettings = function(options)
+    return Global.call("loadSettings", options)
+  end
+
+  -- returns option panel table
+  OptionPanelApi.getOptionPanel = function()
```

Comment:
Let's just make this getOptions() or getPreferences().  getOptionPanel() sounds like you're retrieving the UI element itself